locations/jclouds/src/test/java/org/apache/brooklyn/location/jclouds/JcloudsSshMachineLocationStubbedTest.java

```diff
@@ -118,4 +123,17 @@ public void testWinrmConfigPassedToMachine() throws Exception {
         assertEquals(machine.config().get(WinRmMachineLocation.COPY_FILE_CHUNK_SIZE_BYTES), Integer.valueOf(123));
         assertEquals(machine.config().get(WinRmTool.PROP_EXEC_TRIES), Integer.valueOf(456));
     }
+
+    @Test
+    public void testNodeSetupCustomizer() throws Exception {
+        final String testMetadata = "test-metadata";
+        obtainMachine(ImmutableMap.of(JCLOUDS_LOCATION_CUSTOMIZERS, ImmutableList.of(new BasicJcloudsLocationCustomizer(){
+            @Override
+            public void customize(JcloudsLocation location, NodeMetadata node, ConfigBag setup) {
+                setup.configure(ADDITIONAL_CONNECTION_METADATA, testMetadata);
```

Comment:
Let's also assert that `node` is not null (given it will be, based on the bug I spotted (I believe)).
And might as well also assert that `location` is equal to the object we expect.